packages/backend/src/utils/parsers/quadlet-dryrun-parser.ts

```diff
@@ -39,6 +40,16 @@ export class QuadletDryRunParser extends Parser<RunResult & { exitCode?: number
     return Object.values(this.services);
   }
 
+  // https://www.freedesktop.org/software/systemd/man/latest/systemd.service.html#Service%20Templates
+  protected isTemplate(path: string): boolean {
```

Comment:
This method is not used anymore, it can be safely removed I think

Reply:
https://github.com/podman-desktop/extension-podman-quadlet/pull/752/commits/fbba74ec909eea4f1a5038a6f28e430c8da60feb